File: OCH_Tao/ABC/A/065.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
if (input[1] - input[2] >= 0) {
  console.log("delicious");
} else if (input[0] - (input[2] - input[1]) >= 0) {
  console.log("safe");
} else {
  console.log("dangerous");
}

File: OCH_Tao/ABC/A/065.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  int X, A, B;
  cin >> X >> A >> B;
  if (B <= A)
  {
    cout << "delicious" << endl;
  }
  else if (B - A <= X)
  {
    cout << "safe" << endl;
  }
  else
  {
    cout << "dangerous" << endl;
  }
}